config/rbac/instascale_configmap.yaml

```diff
@@ -0,0 +1,7 @@
+kind: ConfigMap
```

Comment:
Is it needed? It seems like a duplicate?

Reply:
Thanks Antonin, 
Yea, this where we could add it if we want it in the `instascale-1-system`. It will be removed if were happy adding the configmap to the kube-system namespace